local/docker-compose.yml

```diff
@@ -92,8 +92,8 @@ services:
       # - "../html/modules/contrib:/srv/www/html/modules/contrib:rw"
       # - "../html/themes/contrib:/srv/www/html/themes/contrib:rw"
       # Mount nginx custom configuration.
-      # - "../docker/etc/nginx/apps/drupal/drupal.conf:/etc/nginx/apps/drupal/drupal.conf"
-      # - "../docker/etc/nginx/custom:/etc/nginx/custom"
+      - "../docker/etc/nginx/apps/drupal/drupal.conf:/etc/nginx/apps/drupal/drupal.conf"
+      - "../docker/etc/nginx/custom:/etc/nginx/custom"
```

Comment:
Any reason to have this change in that PR? It seems unrelated to the bump of the mariadb version.

Reply:
No, it is unrelated and I can't remember why I needed to change it locally - I've reverted that section.